keps/sig-api-machinery/0006-apply.md

```diff
@@ -194,8 +195,27 @@ atomic. That can be specified with `// +mapType=atomic` or `//
 +structType=atomic` respectively. They map to the same openapi extension:
 `"x-kubernetes-map-type": "atomic"`.
 
-#### Kubectl
-##### Server-side Apply
+#### Clients
+
+##### Apply Status Header
+
+Client side apply in kubectl provides the user with information if the applied config resulted in any changes
+on the apiserver.
```

Comment:
Not sure it really matters, but that's not exactly HOW it happens. The client tries to detect changes and will send these changes if they exist.

Reply:
i made it a little clearer. thanks for this information, didn't know that.